tinywallet/pyproject.toml

```diff
@@ -34,7 +34,7 @@ pytest-cov = "^2.8.1"
 flake8 = "^3.7.9"
 black = "^19.10b0"
 isort = "^4.3.21"
-Cython = "^0.29.16"
```

Comment:
Not sure how I managed to put Cython in the tinywallet dev dependencies...

Reply:
> Not sure how I managed to put Cython in the tinywallet dev dependencies...

Now I know why: if both projects don't have it, the Github action is going to fail.